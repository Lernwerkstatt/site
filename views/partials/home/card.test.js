const card = require("./card.hbs");

describe("card.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      card({
        title: "Aktuelles",
        text:
          "Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis ea consequatur quibusdam sint laboriosam voluptates."
      })
    ).toMatchSnapshot();
  });
});
