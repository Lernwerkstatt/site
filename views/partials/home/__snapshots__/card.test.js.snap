// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card.hbs should match the snapshot 1`] = `
"<div class=\\"card easy\\">
  <div class=\\"card-body\\">
    <h3 class=\\"card-title\\">Aktuelles</h3>
    <p class=\\"card-text\\">
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis ea consequatur quibusdam sint laboriosam voluptates.
    </p>
  </div>
</div>
"
`;
